cmd/agent/api/agent/agent.go

```diff
@@ -38,13 +42,21 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+type contextKey struct {
+	key string
+}
+
+// ConnContextKey key to reference the http connection from the request context
+var ConnContextKey = &contextKey{"http-connection"}
+
 // SetupHandlers adds the specific handlers for /agent endpoints
 func SetupHandlers(r *mux.Router) *mux.Router {
 	r.HandleFunc("/version", common.GetVersion).Methods("GET")
 	r.HandleFunc("/hostname", getHostname).Methods("GET")
 	r.HandleFunc("/flare", makeFlare).Methods("POST")
 	r.HandleFunc("/stop", stopAgent).Methods("POST")
 	r.HandleFunc("/status", getStatus).Methods("GET")
+	r.HandleFunc("/streamLogs", streamLogs).Methods("POST")
```

Comment:
Using `/stream-logs` might be better for URI and we already have this as convention - see `/dogstatsd-stats`